src/util/util.ts

```diff
@@ -45,6 +45,38 @@ function date2input(date: string) {
   return `${month}${day}${year}`;
 }
 
+function datetime2input(date: string) {
```

Comment:
Could you add a comment here detailing format of input and output?

Same for `input2datetime`

Reply:
Done